subsys/portability/cmsis_rtos_v1/cmsis_thread.c

```diff
@@ -179,3 +209,15 @@ osStatus osThreadYield(void)
 	k_yield();
 	return osOK;
 }
+
+osStatus osThreadInstListInit(const osThreadDef_t *thread_def)
```

Comment:
`osThread*` are CMSIS API which are pre-defined. Please rework to avoid adding new APIs.